composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "multitheftauto/mtasa-php-sdk",
+    "description": "PHP Sdk to interact with MTA's server. See more at https://wiki.multitheftauto.com/wiki/PHP_SDK",
+    "type": "project",
+    "license": "GPL-3.0",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.0",
```

Comment:
This should be increased to 7.1.
You have used https://github.com/multitheftauto/mtasa-php-sdk/pull/2/files#diff-74763f9c84d84cc78a22ed38d80d45f9R25 nullable return types, which are introduced with PHP 7.1 :)

Reply:
Yeah, I was trying to make it compatible with PHP 7, didn't know about that. I will change it, thanks

Reply:
Or is it better to replace with false boolean? I mean, there is an important number of people using PHP 7. What do you think?

Reply:
I would increase just the php version to 7.1 and done.
PHP 7.0 security support already left out 😅